src/components/YourFeed.tsx

```diff
@@ -350,11 +355,21 @@ export class YourFeed extends React.PureComponent<DispatchProps & StateProps, Yo
     }
 
     private renderListHeader = () => {
-        return (
-            <FeedHeader
-                navigation={this.props.navigation}
-                onSavePost={this.props.onSavePost} />
-        );
+        if (this.props.displayFeedHeader) {
+            return (
+                <FeedHeader
+                    navigation={this.props.navigation}
+                    onSavePost={this.props.onSavePost}
+                />
+            );
+        } else {
+            return (
+                <NavigationHeader
```

Comment:
Just an idea: we could put an Unfollow button on the right side so that you don't have to go to the settings and find a feed to unfollow.

Reply:
same as above :)